src/mapml/control/LayerControl.js

```diff
@@ -104,10 +108,28 @@ export var LayerControl = L.Control.Layers.extend({
 
     // focus the first layer in the layer control when enter is pressed
     _focusFirstLayer: function(e){
-      if(e.key === 'Enter' && this.className != 'leaflet-control-layers leaflet-control leaflet-control-layers-expanded'){
```

Comment:
The event was on _container before, but I can't understand why the != here didn't prevent _focusFirstLayer from having the desired effect.